source/agora/node/Ledger.d

```diff
@@ -356,6 +361,74 @@ public class Ledger
         return true;
     }
 
+    /***************************************************************************
+
+        Returns the public keys of the validators that signed the block at
+        the specified height, and and also not classified as 'missing' validator
+
+        Params:
+            height = block height
+
+        Returns: the public keys of the validators that signed the block at
+                 the specified height, and and also not
+                 classified as 'missing' validator
+
+    ***************************************************************************/
+
+    private Point[] getPKeysOfSignedNonMissingValidators (Height height) nothrow @safe
```

Comment:
It should not be possible to be both `missing` and signing, as explained under this comment.
Also, perhaps this function could use with a better name, such as `getRewardEligibleValidators` ? It's a more semantic / abstract name, which makes sense *unless* we have a need to get the list of signers for other purposes ?